src/app/models/card.ts

```diff
@@ -0,0 +1,11 @@
+export class Card{
+
+    constructor(
+        public cardId:number,
+        public question:string,
+        public answer:string,
+        public creatorId:number,
+        public stackId:number,
+        public tags:string[],
```

Comment:
This should be tag objects.

Reply:
If you're waiting for Tom to do those, let me know and I'll approve the string array.